app/views/stages/_fields.html.erb

```diff
@@ -175,6 +175,18 @@
 </fieldset>
 
 <fieldset>
+  <legend>Jenkins</legend>
+  <p>Add Jenkins jobs for browser tests that should run after this stage gets deployed.</p>
```

Comment:
can be shortened to `Jenkins jobs that should run after this stage gets deployed.`
also kind very similar to the text below, maybe kill one of them